oauth2_http/java/com/google/auth/oauth2/IamUtils.java

```diff
@@ -142,7 +144,9 @@ private static String getSignature(
                     IamUtils.IAM_RETRYABLE_STATUS_CODES.contains(response.getStatusCode())));
     request.setIOExceptionHandler(new HttpBackOffIOExceptionHandler(backoff));
 
+    LoggingUtils.logRequest(request, LOGGER, "auth sending request to get signature.");
     HttpResponse response = request.execute();
+    LoggingUtils.logResponse(response, LOGGER, "auth received response for signature.");
```

Comment:
nit: Could the message to be more descriptive? i.e. Signature to sign the blob or something that better describes this